examples/basic_example/server.py

```diff
@@ -23,19 +23,31 @@ def get_initial_model_parameters(model: nn.Module) -> Parameters:
 
 
 def fit_config(
-    local_epochs: int,
     batch_size: int,
     current_server_round: int,
+    local_epochs: Optional[int] = None,
+    local_steps: Optional[int] = None,
 ) -> Config:
-    return {"local_epochs": local_epochs, "batch_size": batch_size, "current_server_round": current_server_round}
+    if local_epochs is not None:
```

Comment:
We're repeating this code in a lot of places (which is perfectly fine), but maybe we can pull it into a function and place it in `examples/utils` so all examples can reuse it?